src/main/java/seedu/address/model/Model.java

```diff
@@ -129,7 +129,6 @@ public interface Model {
     /** Returns an unmodifiable list of line chart data for income */
     XYChart.Series<String, Number> getIncomeLineChartData();
 
-    // todo remove updateFilteredEntryList after 'find' command has been implemented
```

Comment:
Removed this comment because we discussed yesterday that we don't need `find` but the method is used so I kept it.